src/grab-vscode-versions.ts

```diff
@@ -19,6 +19,7 @@ export class GrabVSCodeVersions {
 
     static readonly VSCODE_URL_PATTERN_PRE_1_63 = 'https://raw.githubusercontent.com/Microsoft/vscode/${VERSION}/src/vs/vscode.d.ts';
     static readonly VSCODE_URL_PATTERN = 'https://raw.githubusercontent.com/Microsoft/vscode/${VERSION}/src/vscode-dts/vscode.d.ts';
+    static readonly CURRENT_REFERENCE_TARGET_VERSION = '1.55.2';
```

Comment:
@JonasHelming  are we documenting the version we want to cover, or the version we currently cover? I'm not sure this value is important as we often add APIs from multiple versions, and it can higher than `1.55.2` as well. I don't see why knowing this value from the report point of view is important to document.

The actual supported API version is:

https://github.com/eclipse-theia/theia/blob/432602946323bc1500c585de4432021b6d6a4a0a/dev-packages/application-package/src/api.ts#L21

Reply:
This is the current target, we will communicate this more explicitly. Therefore, adding it to the report helps us tracking the gaps.
Yes, we currently work twofold: We fix important gaps reported from newer versions (driven by individual needs). However, we are currently in parallel aiming at increasing the base version which we can only achieve by settings a new target. So it is actually very helpful for us to have it in the report.

Reply:
It might be good to add some additional information in the pull-request description (similarly to what we do in the main repo) and/or the commit, at the moment there is no information. The version we choose as part of https://github.com/eclipse-theia/theia/issues/9863 is arbitrary, in general we add APIs that are not necessarily present in `1.55.2` (I'm not sure how that version was even determined).

In essence, does this change add a new column to the report to see the compatibility of master against `1.55.2`?

Reply:
I added more description. The version 1.55.2 was set as a target by Thomas back then and we aim at completing it now before we move to the next target. You can claim it's arbitrary, but it is reasonably close to completion to make sense and we already have the full details covered in Thomas epic, so it makes sense to us.
There is no official rule preventing you to fix newer API, this is basically what we (ES) plan to do and maybe other will join. The column in the report does support us with this and we will communicate the goal based on this.
BTW, the suggestion to add this was from @tsmaeder , I did it now, because I find it super useful and it would ease our coordination.